appuio/haproxy/templates/deployment.yaml

```diff
@@ -71,13 +71,26 @@ spec:
           volumeMounts:
             - name: haproxy-config
               mountPath: /etc/haproxy/
+            - mountPath: /var/lib/haproxy
+              name: haproxy-script
+            - mountPath: /secrets
```

Comment:
Mount the password as env variable instead of a volume, to have the root password as env variable

Reply:
because script does not have access to env variables, therefore it must be attached as file :/ script is run as subprocess of haproxy and haproxy does not pass envs into it